### PR TITLE
Mechanical Object Section 1

### PR DESCRIPTION
**Pull request by ridetoruin**

### CODE REVIEW

File: src/pages/hegel/reference/The Mechanical Object: Section 1 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# The Mechanical Object – Section 1
```
Comment (Firgrep) [inline src/pages/hegel/reference/The Mechanical Object: Section 1:1]: It looks you've uploaded two nearly identical files. This one should be deleted
image

File: src/pages/hegel/reference/mechanical-object-section-1.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# The Mechanical Object – Section 1
+
+
+The first section opens with a paragraph that is a reflection on the “mechanical object” rather than a continuation of the immanent development of what the “mechanical object” is, as was the case in the previous paragraph. This is something that Hegel does often, and without any warning, and it is important to distinguish passages that add to the immanent development of a concept from passages that are reflections on that immanent development.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: Maybe start with a quote? Given you're probably going to do heavy-duty analysis, it might be helpful for the reader to have something to look at.

"case in the previous paragraph. " What previous paragraph? Your next sentence would make much more sense with a bit more context.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
...ate identity” such that it is a “universality that pervades the particularity and in it is immediate individuality” (711). In other words, it is incongruent with the concept of the “mechanical object” to conceive of it as having the difference of anything as the foundation for its being. What it is for it to be a “mechanical object” to be is for it to be an immediate identity. What is an immediate identity? The determinations of the “Concept” that constitute the “mechanical object”. One cannot conceptually distinguish the universal moment from the individual moment of the “mechanical object” because they are taken as identical to each other. The first sentence of the second paragraph of Section 1 attests to this clearly: “The object is therefore in the first instance *indeterminate*, in so far as it has no determinate opposition in it; for it is the mediation that has collapsed into immediate identity”  (712). The fact that the determinations of the “Concept” are immediately identical to each other and not mediated means that the “mechanical object” is indeterminate. In other words, it lacks determinacy because there is no difference within it for any determinacy to be established.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object-section-1.mdx:6]: 1. "Section 1." => What is Section 1? Are we in "A. Mechanical Object. I." ?
1. “mechanical object” => Why is this in quotation marks?
1. “such an abstract difference of individuality and universality is excluded by the Notion of object”  (712). => What citation system are you using here? This is something we should agree on sooner than later...
1. "To answer this question we need only to cast our minds back to the first paragraph of “Mechanism”," => was this anywhere discussed?
1. "immediate identity? The determinations of the “Concept” that constitute the “mechanical object”." => is this right? `immediate identity` sounds more like essence-talk rather than concept. I see that you go into explaining how it relates to the concept but it may be better to be up front that this relates to the concept rather than essence. 

Other than that, great paragraph! I particularly like the example with the rock, that's solid ;)

File: src/pages/hegel/reference/mechanical-object-section-1.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
...– they do not relate to anything outside of themselves and are only self-relating. It is in this sense that the “mechanical object” is both indeterminate and has difference within it. It has difference within it because of its determinateness, because of its negative relation to itself. In this moment of negative self-relation, the “mechanical object” is immediately identical to all three determinations of the “Concept”. It is indeterminate because it does not relate to anything outside of itself, it is only self-relating. In its self-relating it is, effectively, relating to itself as the immediate identity of the determinations of the “Concept”. It is for this reason that Hegel writes the following: “Because this indeterminate determinateness is essential to the object, the latter is within itself a *plurality* of this kind, and must therefore be regarded as a *composite* or *aggregate*” (712).  What it is for the “mechanical object” to be is for it to relate to itself as the immediate identity of other objects (the determinations of the “Concept”). In other words, what it is for it to be itself is for it to be itself through other objects that are taken to be identical to itself.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object-section-1.mdx:10]: 1. Long, heavy paragraph. Can it be chopped up? I think you do a great job working through the material, but it is a tough read still. 
2. “Doctrine of the Concept” => does this need quotation marks?
3. "The ‘universality’ of a determination is its core essence – it is what it is for that determination to be what it is." => loosely speaking, I kind of see what you're getting at, but when looked at these terms technically, `essence` and `what is` (which seems to signify `being`) don't seem to cohere.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
..., for these are not objects because they are not totalities” (712).  It is worth musing on this brief remark because it reveals a lot about how Hegel conceptualises the wider significance of his account of the “mechanical object”. Hegel clearly has some mechanistic, in particular atomistic, worldviews in his crosshairs when he goes out of his way to state that the “mechanical object” does not consist of atoms. Hegel’s target is those that theorise that the world is fundamentally made out of atoms and that since the world is made out of atoms that everything is really just atoms.  In Hegel’s account of “Mechanism”, he goes out of his way to explicitly state that the “mechanical object”, the simplest kind of mechanical entity, is not reducible to atoms. There is not something more fundamental to the “mechanical object” that is the true reality of things. Moreover, and perhaps a stronger rebuke to the atomists, not only is the “mechanical object” not reducible to atoms, but it is also not the most fundamental thing. Hegel rejects this notion of fundamentality out of hand and it is this rejection that is his strongest argument against atomistic or atomistic-like conceptions of reality.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object-section-1.mdx:12]: 1. "theorise that the world is fundamentally made out of atoms and that since the world is made out of atoms that everything is really just atoms" => sure, but is the `world` really at question here? The real target is the nature of mechanical object. I suppose that you could say if the mechanical object as such cannot be understood as mere atoms, the neither can the world if it is only made up of mechanical objects. 
2. About the issue of "fundamentality". Even if one grants that reality isn't fundamentally made up of atoms, one could still argue that the mechanical object is fundamentally made up of atoms (and thereby assume that something else more fundamental is the further "bottom of reality"). I think Hegel wants to say that no matter the fundamental aspect of reality/world/etc, the mechanical object is not reducible to atoms. Regardless, can we mount a better support of Hegel's claim here, if at all possible?

File: src/pages/hegel/reference/mechanical-object-section-1.mdx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+What follows from the last quotation is a long remark about how Hegel’s account of the “mechanical object” is akin to Leibniz’s conception of a monad. Hegel writes:
+
+>“The Leibnizian *monad* would be more of an object since it is a total representation of the world, but confined within its *intensive subjectivity* it is supposed at least to be essentially *one* within itself. Nevertheless, the monad determined as an *exclusive one* is only a principle *that reflection assumes*. Yet the monad is an object, partly in that the ground of its manifold representations-of the developed, that is, the *posited* determinations of its merely *implicit* totality-lies *outside it*, and partly also in that it is indifferent to the monad that it constitutes an object *along with others*; it is thus in fact not *exclusive* or *determined for itself*” (712).
+
+**Here, I invite a scholar with knowledge of Leibniz and his monads to expound on the above paragraph.**
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object-section-1.mdx:18]: Do you want to explicitly leave this like that? I think putting a generic "this section is a stub, you can help us improving it by contributing to it" kind of thing, like wikipedia does. What do you think?

**Review by Firgrep — CHANGES_REQUESTED** [review]

**Comment by Firgrep** [comment]
Moreover, if you're planning on making multiple files about the mechanical object (looks that way), then you should make a folder `mechanical-object` put your files there. You'll likely need a `_meta.json` file to appropriately arrange the files (the website builder does this alphabetically automatically, but you want to override that). It's quite simple to set up, so let me know when you want to spend some time on it.

File: src/pages/hegel/reference/The Mechanical Object: Section 1 (hunk 1 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/The Mechanical Object: Section 1:1]: I chose to do a break-down of my articles in accordance with the way that Hegel breaks up the chapter. So, in Mechanism, he begins with that five line preliminary paragraph where he introduces the mechanical object. That is the focus of the published article "The Mechanical Object First Paragraph". I chose to do this because (a) this is how I generally approach breaking-down Hegel's text and (b) I think it's a helpful way to follow the text because Hegel's break-down has a rationale behind it that makes the logical development easier to follow.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 2 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: Would you suggest quoting the full paragraph at the start? I'd be happy to do that.

In this case, the "previous paragraph" refers to the first paragraph of Mechanism that is treated in the first article of the Mechanism series.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 3 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:6]: 1. I put "Section 1" because this section of Mechanism is begun with a number "1" at the beginning of the paragraph. After that there's a "Section 2" and a "Section 3". 
2. That's just something I carried over from my personal approach. It's a way of distinguishing when I'm writing about the determination within the logical development and when I'm writing about the determination in more general terms. I understand that that's a pretty big stylistic feature to implement. So if you think it's too much, I'm happy to drop it. 
3. I was trying to keep it simple. We all know that it's the Science of Logic - though it should probably state somewhere that it's Miller's translation- and so I just put the page number. That said, I do think it's probably a good idea to always footnote the original German when citing so as to have some original text to anchor interpretations so that we don't get involved in disagreements that arise out of translation discrepancies. 
4. This is a reference to the first article of the Mechanism series. Would probably be good to add a hyperlink whenever I refer to it.
5. That should be clear from what I write in the first article. I explain that the immediate identity is about the Concept. Let me know if you think that's unclear.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 4 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:10]: 1. Yep, can definitely break it into 3 paragraphs.
2. Again, I do that as a way of distinguishing between logical use and general use. Refer to point 2 in the above comments box - happy to drop this stylistic feature.
3. I guess that's an interesting point. One thing to say is the following, isn't the Concept the unity of Being and Essence? Not sure, what to make of that in terms of the above, but that's the first thing that came to mind. 
More loosely, I use the "essence" language because Hegel refers to it as an "essential determinateness" and I use the "being" language because this is ontology, so at every step we are thinking about what it is for a particular determination to be. I have this itch that there is a bigger discussion to be had about this, but I can't quite pinpoint it.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 5 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:12]: 1. Well, insofar as atomistic theories are theories about reality, I think that Hegel is definitely saying something about reality or the world when he is remarking that the mechanical object is not made up of atoms. Moreover, isn't he saying something about the world all the time in the SL? If it is a study of the fundamental determinations of Being and Thought, then our understanding of each of these determinations goes someway towards our understand of the world or reality.
2. Well, first of all, I think that Hegel's argument against atomism can be more generally captured by his argument against fundamentality, which is what I try to explain in the final sentence. I suppose I could say more about this, but my concern would be that it might take us too far away from Mechanism. Could be hyperlinked to a Guide article about fundamentality? But it is this more general stance against fundamentality that is Hegel's proper attack on a theory like atomism. 
I think he mentions it here because we are in Mechanism and atomism is a mechanistic worldview. I think you're right to say that Mechanism itself is not a rebuke to atomism. It is a rebuke in the sense that Hegel's account of Mechanism does not permit an atomistic theory of the world, but it is not a direct attack on atomism. That attack must be found in his critique of foundationalism.
So to summarise, I can edit the final sentences to better express the point. Thanks!

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 6 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:18]: I think it depends on the tone of the article and, perhaps, the website overall. What I wrote was meant to create a tone of community and also to add an individual touch by specifying what I think needs to be added here. What you're suggesting is more down-the-line and impersonal. 
I like what I've done, but if you think it opens the door to too much interpretation, happy to alter it.

**Comment by ridetoruin** [comment]
Thanks a lot for the above. I'll wait for your responses to some of the above and I'll make the edits.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 2 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: Yeah, either put up a quote or do a link to it. To make a link, you follow this pattern: 

```
[displayed name](/path/from/root/to-your-article.mdx)

// e.g.
[Hegel's Being](hegel/reference/being)
```

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 3 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:6]: 1. I think will be sorted by the folder structure you set up. ✅
2. Well, if you mean to target the category in a technical sense, we can actually make use of the `code` formatting. It's what programmers use to highlight pieces of code. Since our texts will have no code to highlight, but we will highlight something technical, I thought it could be cool to appropriate it for ourselves. `code` formatting puts a text in courier font with a shaded background. I've used it myself in the writing I've done and it makes sense and looks good. You can make `code formatted text` by enclosing any bit with backticks " ` " <-- that thing in the middle there. 
Another cool thing is formatted code blocks. I've been using these to highlight a quotation with technical intent. You start a line with three backticks and then end the bit with another three with a new line at the end.
```md
In its indeterminate immediacy it is equal only to itself
and also not unequal with respect to another;
it has no difference within it, nor any outwardly.
```
What do you think?

3. We need to agree on a citation system. The encyclopaedia should adhere to one standard. I think MLA looks and works fine with inlined references. I think we should avoid using footnotes as much as possible, since they will be very hard to maintain (we don't enjoy an automatic footnote system here right now). 

4. Yes, either a link or briefly include the point. 
5. Ok ✅

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 4 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:10]: 1 and 2 ✅

3. Ok, we'll leave it for now and can open a discussion later on it. ✅

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 5 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:12]: 1. Saying that SL says something about the world is a little imprecise given that the `world` should have some sort of categorical definition, but it doesn't. I think it's fine to refer to the world loosely to help our understanding, but to use it as a technical leverage is thin since the concept is pretty vague. In one sense, mechanism does address this vagueness since it aims to determine a grand totality coherently unto itself. 
2. Ok, thanks ✅

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 6 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:18]: The issue is that it appears like a part of the article itself, which it shouldn't be. Moreover, putting it in bold is too striking. I agree that we could do something more personal, so we can use this component (see image below) as a template and then whoever can just add their own bit of text.

image

File: src/pages/hegel/reference/mechanical-object/section-2.mdx [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
...ifference that explains how a moment can be both a determinateness and indeterminate in that determinateness. As Hegel writes, the “mechanical object” is “indifferent to the determinations as individual, as determined in and for themselves, just as the latter are themselves indifferent to one another” (713). As such, since each determination is indifferent to every other determination, no determination can be comprehended from any other determination. The idea being that if a does not actively relate to b (does not posit b) then it would not make sense to try to understand b by understanding a. To use an empirical example, if you have two rocks, Ra and Rb, it would not make sense to seek to understand Rb by trying to understand Ra since what it is for Ra to be is for it to be indifferent to Rb: Ra is perfectly able to be itself without any explicit reference to Rb and any reference that there is to Rb is merely a formal one¸ i.e. merely in virtue of the fact that both rocks happen to be in the same place. This is what Hegel means when he concludes the first paragraph of Section 2 with a clarification of how the “mechanical object” can have both determinateness and be indeterminate:
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/section-2.mdx:7]: You must have 1 line between paragraphs in markdown in order to designate paragraphs.

File: src/pages/hegel/reference/mechanical-object/section-2.mdx [hunk 8]
```diff
@@ -0,0 +1,24 @@
+---
+title: Section 2
+description: Learn about the second section of the Mechanical Object
+---
+
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/section-2.mdx:5]: Some headings would be nice

**Review by Firgrep — CHANGES_REQUESTED** [review]
See new comments

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 2 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: What would the link be connected to?

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 3 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:6]: 2. That sounds like a very nice function. I'll look to implement it in the article. 
3. Yes, MLA is easy-peasy. We can avoid footnotes, but we will need some acceptable way of making notes within the article. Maybe we created indented paragraphs in bold?
4. I think a link would be sleeker.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 2 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: What do you mean? You make a link to the previous section? Or do you mean on this article? You could use the word paragraph, since that's what you're referring. Any word can be made a link, whatever's most intuitive. `[paragraph](/my/folder/file)`

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 3 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:6]: 3. There is footnote functionality, it's just that it should be used sparingly because we will need to manually update potentially all footnotes whenever there's a change. 
But I also wonder if one needs to go on lengthy tirade about something in a footnote, then either that should be important enough for the main body or be separated out into a new article or discussion (which one could link to). This is something we can embrace instead.

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 2 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: So just to clarify, by following the above, I would be linking it to a previous article?

Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: A link only links exactly where you tell it to. 

Here is a normal, full length link (full length because it includes the https at start). This is what you usually do when you share links externally.

https://www.youtube.com/watch?v=mFjU4JuJgnM

Here is the same link, but formatted with a different display text.

[Click me, baby!](https://www.youtube.com/watch?v=mFjU4JuJgnM)

When you are building a website that has a router, like ours do, we can use _relative links_ . We can drop the https and the domain name and focus only on the path starting with "/", which is known as the root and is usually the home page. Any subsequent words and slashes after "/" define a path, such as "/hegel" or multiple like "/hegel/reference" etc. Have a look at the demo website to see this in action.

Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: I've added the block quote at the start of the page. 
 
Regarding the link, would it be like this: src/pages/hegel/reference/mechanical-object.mdx?

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 3 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:6]: OK. I've done the MLA referencing, and I've done the three ticks addition, too!

File: src/pages/hegel/reference/mechanical-object/section-2.mdx (hunk 8 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object/section-2.mdx:5]: done!

File: src/pages/hegel/reference/mechanical-object-section-1.mdx (hunk 2 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object-section-1.mdx:4]: No, the `src/pages` is not part of the URL.

The URL "begins" inside of the `pages` folder. 

Also, you drop extensions `.mdx` for URLs.

**Review by Firgrep — APPROVED** [review]
